build/VSL.Imports.Closed.targets

```diff
@@ -70,8 +70,8 @@
           Inputs="@(IntermediateAssembly)"
           Outputs="@(IntermediateAssembly);$(PostCompileBinaryModificationSentinelFile)">
     
-    <Exec Command="$(VSLOutDir)\FakeSign.exe &quot;@(IntermediateAssembly)&quot;" />
-    
+    <Exec Condition=" '$(OS)' == 'Windows_NT'" Command="$(VSLOutDir)\FakeSign.exe &quot;@(IntermediateAssembly)&quot;" />
+    <Exec Condition=" '$(OS)' != 'Windows_NT'" Command="mono $(VSLOutDir)\FakeSign.exe &quot;@(IntermediateAssembly)&quot;" />
```

Comment:
I actually tried to implement `NetExec` at one point and it turned out to be significantly more complex than I originally expected.  Eventually I ended up taking the following approach:

https://github.com/dotnet/roslyn/commit/3cef72ec8ed4d8dae6c2210f58ae521927aace6a#diff-9d3884f8609197f7e7500f5d4726077f

It's not nearly as elegant but it works and keeps the number of duplicate elements due to `Conditional` low.